bundle/manifests/apps.3scale.net_apicasts.yaml

```diff
@@ -38,13 +38,14 @@ spec:
             description: APIcastSpec defines the desired state of APIcast
             properties:
               adminPortalCredentialsRef:
-                description: LocalObjectReference contains enough information to let you locate the referenced object inside the same namespace.
+                description: Secret reference to a Kubernetes Secret containing the portal endpoint URL information. The Secret must be located in the same namespace
                 properties:
                   name:
                     description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
Seems you missed this one?

Reply:
Unfortunately we can't control that one. That field `Name` is part of the `v1.LocalObjectReference` K8s type (see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#localobjectreference-v1-core).
The description texts are taken from comments in the source code. The source code belongs to K8s so we don't have control over the descriptions shown for the attributes of the fields that are K8s objects.